hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/SCMStateMachine.java

```diff
@@ -175,6 +176,14 @@ private Message process(final SCMRatisRequest request) throws Exception {
     }
   }
 
+  @Override
+  public void notifyLogFailed(Throwable ex,
+                              RaftProtos.LogEntryProto failedEntry) {
+    // Should we add retry here?
+    LOG.error("SCM statemachine appendLog failed, entry: {}", failedEntry);
+    ExitUtil.terminate(1, ex);
```

Comment:
```suggestion
    ExitUtils.terminate(1, ex.getMessage(), ex, StateMachine.LOG);
```

Reply:
Thx for @JacksonYao287, I have replaced it with your suggestion.